openlibrary/templates/lists/widget.html

```diff
@@ -468,10 +468,8 @@ <h2>Create a new list</h2>
     }
 
     function setup_events() {
-        \$("#create-new-list").click(function() {
-           close_popup();
-           \$.colorbox({inline: true, opacity: "0.5", href: "#addList"});
-        });
+        \$("#create-new-list")
```

Comment:
This change is also significant - since this template is loaded on most pages - that also means jquery ui is now loaded on every page - and worse - it's loaded via a new HTTP request rather than as part of the initial payload. The motivation of the patch was to delay or avoid the load of jquery ui on all pages.

The click handler made sure the popup was only rendered and displayed when the element is clicked and the intent is there.

The commit message "Load on init instead of on user interaction to fix some bugs:"  goes against the ethos of this patch  - the whole point was to load on user interaction to clear out the critical path.

I am happy to merge this one provided the above length check if fixed and we're committed to understanding this bug better and fixing ASAP.